code/modules/spacepods/equipment.dm

```diff
@@ -49,23 +49,35 @@
 	var/obj/item/device/spacepod_equipment/weaponry/weapon_system // weapons system
 	var/obj/item/device/spacepod_equipment/misc/misc_system // misc system
 	var/obj/item/device/spacepod_equipment/cargo/cargo_system // cargo system
+	var/obj/item/device/spacepod_equipment/cargo/sec_cargo_system // secondary cargo system
 	var/obj/item/device/spacepod_equipment/lock/lock_system // lock system
-	//var/obj/item/device/spacepod_equipment/engine/engine_system // engine system
-	//var/obj/item/device/spacepod_equipment/shield/shield_system // shielding system
 
 /datum/spacepod/equipment/New(var/obj/spacepod/SP)
 	..()
 	if(istype(SP))
 		my_atom = SP
 
+/datum/spacepod/equipment/proc/get_list()
+	var/list/stuff
+	for(var/obj/I in vars)
```

Comment:
There should be a better way to do this than looping through `vars`. Since you can't just loop through `contents`, you might want to have a list in the equipment datum, and have spacepod equipment be added to that list on being equipped to the pod, and removed upon being removed from the pod. You do need to make sure that everything GCs cleanly if you do that though.